src/openeo_odc/map_to_odc.py

```diff
@@ -52,7 +54,18 @@ def resolve_from_parameter(node):
             # Argument is not iterable (e.g. 1 or None)
             continue
         if 'from_parameter' in node.arguments[argument]:
-            in_nodes[argument] = node.parent_process.arguments['data']['from_node']
+            if argument == 'data':
+                in_nodes['data'] = node.parent_process.arguments['data']['from_node']
+                try:
+                    in_nodes['parameters'] = node.parent_process.arguments['parameters']
+                except:
+                    continue
+            elif argument == 'x':
+                in_nodes[argument] = node.parent_process.arguments['data']['from_node']
+            elif argument == 'x':
+                in_nodes[argument] = node.arguments['x']['from_parameter']
```

Comment:
That's the one thing, I wanted to talk with you about. The first line is the old version, the second line is the solution I found, but I am not sure about... it could as well be commented out....